.ci/environments/quarkus-3/patches/0006_jobs-service.patch

```diff
@@ -0,0 +1,581 @@
+commit fda0360631ae97f715be44e61f4a84e0e0305a9c
```

Comment:
@pefernan I don't see any update on any pom.xml.
is the mutiny-zero library transient ?

Reply:
@radtriste yea it is a transient dep, but I can add it to the pom.xml's if you think it's going to be better